src/stock-items/add-stock-item/packaging-units/packaging-units.component.tsx

```diff
@@ -119,54 +121,56 @@ const PackagingUnits: React.FC<PackagingUnitsProps> = ({
     );
 
   return (
-    <FormProvider {...packageUnitForm}>
-      <DataTable
-        rows={[...items, {}]}
-        headers={tableHeaders}
-        isSortable={false}
-        useZebraStyles={true}
-        render={({ headers, getHeaderProps, getTableProps }) => (
-          <TableContainer className={styles.packagingTableContainer}>
-            <Table {...getTableProps()} className={styles.packingTable}>
-              <TableHead>
-                <TableRow>
-                  {headers.map((header) => (
-                    <TableHeader
-                      {...getHeaderProps({
-                        header,
-                        isSortable: false,
-                      })}
-                      style={header.styles}
-                      key={header.key}
-                    >
-                      {header.header?.content ?? header.header}
-                    </TableHeader>
+    <>
```

Comment:
do we really need this tag

Reply:
> do we really need this tag

This has been addressed in the subsequent commit